sdk/resourcemanager/gulpfile.js

```diff
@@ -144,7 +147,15 @@ function codegen(project, cb) {
     }
 
     console.log('Command: ' + cmd);
-    return execa(cmd, [], { shell: true, stdio: "inherit" });
+    autorest_result = execa.sync(cmd, [], { shell: true, stdio: "inherit" });
+
+    // move generated samples to azure-resourcemanager
+    generatedSamplesSource = path.join(mappings[project].dir, '/src/samples/java/', packagePath, 'generated');
+    generatedSamplesTarget = path.join('azure-resourcemanager/src/samples/java/', packagePath);
```

Comment:
do we need /generated folder?

Reply:
Want to have it so it is clear that these are pure generated, and might not have guarantee on its quality.